src/enact/references.py

```diff
@@ -242,19 +263,39 @@ def register_type(self,
                     attributes: Dict[str, Optional[types.TypeDescriptor]]):
     """Register a new type to allow resource to be committed."""
 
+  async def register_type_async(
+      self,
+      type_key: types.TypeKey,
+      attributes: Dict[str, Optional[types.TypeDescriptor]]):
+    """Register a new type to allow resource to be committed."""
+    return self.register_type(type_key, attributes)
+
   @abc.abstractmethod
   def get_type(self, type_key: types.TypeKey) -> (
       Optional[Dict[str, Optional[types.TypeDescriptor]]]):
     """Returns the type, if known."""
 
+  async def get_type_async(self, type_key: types.TypeKey) -> (
+      Optional[Dict[str, Optional[types.TypeDescriptor]]]):
+    """Returns the type, if known."""
+    return self.get_type(type_key)
+
   @abc.abstractmethod
   def commit(self, ref_id: str, packed_resource: PackedResource):
     """Stores a packed resource."""
 
+  async def commit_async(self, ref_id: str, packed_resource: PackedResource):
+    """Stores a packed resource."""
+    return self.commit(ref_id, packed_resource)
```

Comment:
Why return if it doesn't return a value? Or perhaps the return value type annotation is missing?

Reply:
Fixed.